app/views/musees/show.html.erb

```diff
@@ -1,18 +1,23 @@
 <div id="map" style="height: 400px;"></div>
+
+
+<script src="https://api.mapbox.com/mapbox-gl-js/v2.5.0/mapbox-gl.js"></script>
+<link href="https://api.mapbox.com/mapbox-gl-js/v2.5.0/mapbox-gl.css" rel="stylesheet">
+
 <script>
-  function initMap() {
-    var location = { lat: <%= @musee.latitude %>, lng: <%= @musee.longitude %> };
-    var map = new google.maps.Map(document.getElementById('map'), {
-      zoom: 15,
-      center: location
-    });
-    new google.maps.Marker({
-      position: location,
-      map: map
-    });
-  }
-</script>
-<script async defer
-  src="https://maps.googleapis.com/maps/api/js?key=<%= ENV['GOOGLE_MAPS_API_KEY'] %>&callback=initMap">
-</script>
+  mapboxgl.accessToken = "<%= ENV['MAPBOX_API_KEY'] %>";
 
+  // Création de la carte
+  var map = new mapboxgl.Map({
+    container: 'map', // ID du conteneur pour la carte
```

Comment:
instead of

```
container: 'map'
```

it's a good practice to add context to your container 

```
container: 'js-map'
```
that way you know your ID is hooked to a javascript file